### DISCUSSION TITLE
How to configure XRD to launch Plasma instead of XFCE?

### DISCUSSION THREAD

**Discussion by sharpninja** [WhitewaterFoundry/Pengwin#773] [Q&A] [answered]
I would like to use Plasma instead of XFCE as my full screen Pengwin desktop.  How do I configure the launch icon for XFCE to use Plasma instead?

**Comment by sharpninja** [comment]
I found the solution here:

https://askubuntu.com/questions/1237288/20-04-xrdp-kde-plasma-connect-issue

**Reply by crramirez** [reply, marked as answer]
Ah I didn't your answer before commenting. Yes, this is a good answer the change is `echo "/usr/bin/startplasma-x11" > ~/.xsession`

**Comment by crramirez** [comment]
Hello @sharpninja 

The easiest way is to uninstall XFCE. The one that allows you to have both installed, give me some hours and post here the solution, it is a change in a script.

Regards

**Comment by crramirez** [comment]
I haven't tested Plasma with Pengwin for a long time. Could you please post here about your experience? Plasma doesn't requires SystemD but with it, Plasma runs a lot better

**Comment by sharpninja** [comment]
Plasma is great on Pengwin.  Only issue I've run into is that Discover think's there's no network connection.
